controllers/userCtrl.ts

```diff
@@ -0,0 +1,74 @@
+import { Request, Response } from 'express'
+
+import { PrismaClient } from '@prisma/client'
+
+const prisma = new PrismaClient();
+
+export const getAllUser = async (req: Request, res: Response) => {
+    try {
+        const obj= filtering(req.body);
+        console.log("Object --> ",obj)
+        const data = await prisma.user.findMany({skip:(req.body.skip -1) * req.body.take ,take:req.body.take , where: obj,include:{groupOwned:true} });
```

Comment:
Add a where clause instead of calling filtering function

Reply:
Ok corrected.